phpBB/phpbb/textformatter/s9e/parser.php

```diff
@@ -56,24 +50,20 @@ public function __construct(\phpbb\cache\driver\driver_interface $cache, $key, \
 
 		$this->dispatcher = $dispatcher;
 		$this->parser = $parser;
-		$this->user = $user;
 		$parser = $this;
 
 		/**
 		* Configure the parser service
 		*
 		* Can be used to:
-		*  - toggle features according to the user's preferences,
-		*  - toggle BBCodes according to the user's permissions,
```

Comment:
Actually I shouldn't say that one couldn't toggle BBCodes without the user object being passed in the event. It's always possible to toggle BBCodes using the global user object. That's what extensions do in 3.1 and it's still possible in 3.2. I don't know whether that's the correct way to do it though.